audit/projects/k8s-artifacts-prod-bak/services/pubsub/topics.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "name": "projects/k8s-artifacts-prod-bak/topics/container-analysis-occurrences-v1"
```

Comment:
We're now exporting pubsub topics as a result of https://github.com/kubernetes/k8s.io/pull/2133

I believe these to be expected for the project